perfkitbenchmarker/benchmarks/mesh_network_benchmark.py

```diff
@@ -113,6 +114,8 @@ def RunNetperf(vm, benchmark_name, servers, result):
 
   match = re.findall(r'(\d+\.\d+)\s+\n', output)
   value = 0
+  if len(match) != (len(servers) - 1) * FLAGS.num_connections:
+    raise errors.Benchmarks.RunError('Netserver not started on remote vm.')
```

Comment:
Is netserver failing to start the only possible cause of errors here?

Please include the expected number of measurements and the number found in the error message.